### PR TITLE
Update About Us section

### LINKED ISSUE #57: Update list of team members
It would be nice to have a more current Team description, while still giving credit to alumni.

>The Team:
>Arne, Ben, Bettina, Franziska, Jelle, Julia, Lisa, Malwine, Martin & Nicola.

### PR DESCRIPTION
**Pull request by daiyi**
Current team member updates! I pulled images off github/twitter, and anna's from slack. 

features: 
- updated team members
- alumni section <3
- all members are randomised with each page upload with javascript
- graceful fallback when javascript is disabled

ACTIONABLE ITEMS:
- Please let me know what public url you'd like to be associated with your image. I didn't want to assume we all use twitter (: 
- Let me know if you'd like a different image!
- update yourself here: https://github.com/clojurebridge-berlin/organization/wiki/Orga-Team-History

closes #57 

about us section updates

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -20,3 +20,61 @@ url: "http://clojurebridge-berlin.github.io" # the base hostname & protocol for
...
+    url: http://lislis.de
+  - name: Nils
+    image: nils.png
+    url:
+  - name: Saskia
+    image: saskia.jpg
+    url:
+  - name: Tibor
+    image: tibor.jpg
+    url:
```
Comment (codebeige) [inline _config.yml]: Please add `http://codebeige.net` Thank you!

**Comment by JohannaCarola** [comment]
@daiyi you can link my twitter account https://twitter.com/JohanannC 
thank you! <3

File: _config.yml [hunk 2]
```diff
@@ -20,3 +20,61 @@ url: "http://clojurebridge-berlin.github.io" # the base hostname & protocol for
...
+  - name: Dave
+    image: dave.png
+    url: http://daveliepmann.com
+  - name: Johanna
+    image: johanna.jpg
+    url:
+  - name: Julia
+    image: julia.jpg
+    url:
```
Comment (Realtin) [inline _config.yml]: https://twitter.com/anderspree for me! ✨

File: _config.yml [hunk 3]
```diff
@@ -20,3 +20,61 @@ url: "http://clojurebridge-berlin.github.io" # the base hostname & protocol for
...
+  - name: Lisa
+    image: lislis.png
+    url: http://lislis.de
+  - name: Nils
+    image: nils.png
```
Comment (nblumoe) [inline _config.yml:47]: http://nils-blum-oeste.net/ please :) Thank you!

File: _config.yml [hunk 4]
```diff
@@ -20,3 +20,61 @@ url: "http://clojurebridge-berlin.github.io" # the base hostname & protocol for
...
+    url:
+  - name: Saskia
+    image: saskia.jpg
+    url:
```
Comment (saskali) [inline _config.yml]: http://saskialindner.com for me, thank you!

**Review by daveliepmann — APPROVED** [review]

File: _config.yml (hunk 4 above)
Comment (daiyi) [reply _config.yml]: omg I love it!! gonna have to got that flowing hand tree image for you soon (:

**Comment by vsmart** [comment]
AWESOME <3

**Review by vsmart — APPROVED** [review]

File: _config.yml [hunk 5]
```diff
@@ -20,3 +20,61 @@ url: "http://clojurebridge-berlin.github.io" # the base hostname & protocol for
...
+team:
+  - name: Ben
+    image: ben.png
+    url: https://twitter.com/socksy
+  - name: Franka
+    image: franka.jpg
+    url: http://franka.tech
+  - name: Danielle
+    image: dani.jpg
+    url:
```
Comment (endocrimes) [inline _config.yml]: Linking to twitter.com/dantoml would be 💯 ✨ - thanks!

**Comment by daiyi** [comment]
Gonna merge, will make further changes upon request!

**Pull request merged**